omero/autogen_docs

```diff
@@ -44,16 +44,20 @@ if [ -d "$DIRECTORY" ]; then
 fi
 
 mkdir -p $DIRECTORY/osx/
+#generate walkthrough for all os
+ALL=true bash $WORKSPACE/omero-install/autogenerate.sh
```

Comment:
This is the line that seems to have upset the merge-docs-autogen build - needs /linux/ adding to the file path?